test/doc/core/saturated/abs.cpp

```diff
@@ -9,7 +9,7 @@ int main()
   wide_it pi = {-1, 2, -3, -32768};
 
   std::cout << "---- simd" << '\n'
-            << "<- pi                  = " << pi << '\n'
-            << "-> saturated(abs)(pi)  = " << eve::saturated(eve::abs)(pi) << '\n';
+            << "<- pi                       = " << pi << '\n'
```

Comment:
Do ou want to actually redo these docs as tests, since you touch them?

Reply:
I will yes